### PR TITLE
fix(translation): fix spanish translation

### PR DESCRIPTION
**Pull request by pixelead0**
Refs #439

Edited by lex111: we must not close issue #439, because not all translations are exists, and this issue will be open throughout October, while Hacktoberfest passes.

### CODE REVIEW

**Comment by FlorianTaut** [comment]
@pixelead0 Thank you very much!

But shouldn’t that be `es` in line 1?

File: src/locale/languages/es.js [hunk 1]
```diff
@@ -4,319 +4,324 @@
...
-      contactTitle: 'CONTACT',
+        "A cada contribución a la aplicación, se realiza una revisión de código para evitar que alguien agregue cualquier tipo de código malicioso.",
+      contactTitle: "CONTACTO",
       contact1:
-        'Thank you for reading our Privacy Policy. We hope you enjoy using GitPoint as much as we enjoyed building it.',
+        "Gracias por leer nuestra Política de Privacidad. Esperamos que te guste usar GitPoint, tanto como te gusta desarrollarlo",
       contact2:
-        'If you have any questions about this Privacy Policy or GitPoint in general, please file an issue in the',
-      contactLink: 'GitPoint repository',
+        "If you have any questions about this Privacy Policy or GitPoint in general, please file an issue in the",
```
Comment (FlorianTaut) [inline src/locale/languages/es.js]: You missed to remove this line.

Because of this, the travis build fails.

**Comment by lex111** [comment]
@pixelead0 thank you for your work, but we already have PR for translating into Spanish - #442 . Could you check it?

**Comment by pixelead0** [comment]
How can I change or merge the es.js file from pr # 442?

**Comment by lex111** [comment]
@pixelead0 You can not do this, just leave a comment there if need to correct the translation.

**Comment by alejandronanez** [comment]
I think we should wait for #422 before merging this one.
Thoughts @lex111 @FlorianTaut @housseindjirdeh @andrewda

**Comment by lex111** [comment]
@alejandronanez but is there any point in merging this PR?

**Comment by alejandronanez** [comment]
@lex111 actually we have two PRs with almost the same content. What about merging #422 and then see if this one is still relevant?

**Comment by lex111** [comment]
@alejandronanez :ok_hand:

**Comment by geekfred** [comment]
I don't get it. I've checked the es file and there's many, many strings yet to be translated, and found some syntax improvements to be done. Is this being done or is it a finished file? Thanks.

**Comment by alejandronanez** [comment]
@geekfred we already have a PR open before this one with spanish translations https://github.com/gitpoint/git-point/pull/442 we should wait for that PR to get merged first and then see if this one is still relevant.

**Comment by geekfred** [comment]
Excellent!

**Comment by housseindjirdeh** [comment]
@pixelead0 Thanks so much for this. #442 should be merged in soon and I would love if we can cross check any remaining/fixable translations after that's in. Can't tell you much how we appreciate you taking the time to put this PR up 🙌

**Comment by pixelead0** [comment]
I resolved conflicts with #442

**Comment by housseindjirdeh** [comment]
@pixelead0 thank you <3

Did you rebase from latest master because it still seems like a few conflicts are present. Could you take a look whenever you get the chance :pray:

**Comment by machour** [comment]
Hola @pixelead0 !

Spanish translations have been merged a while ago:
   https://github.com/gitpoint/git-point/blob/master/src/locale/languages/es.js

Could you give them a review and tell us if there are still changes to be made? 🙏

**Comment by machour** [comment]
Hi there. Our translations files structure completely changed. I'm going to close this PR for now.
Feel free to open a new PR with fixes, #442 have been merged since then.

Thank you again for all your hard work!

**Pull request closed without merging**